src/data/faq.json

```diff
@@ -158,11 +158,23 @@
                         ]
                     },
+                    {
+                        "title": "Why does the Corona-Warn-App on version 2.8 show that my certificate couldn't be validated although the previous version showed my certificate as valid?",
+                        "anchor": "dcc_validity_check_change",
+                        "active": true,
+                        "textblock": [
+                            "With version 2.8 the behaviour of the certificate validity check has changed. Entry rules for travellers have been set up by all EU member countries, but until today only a few countries provided entry rules in machine-readable format which Corona-Warn-App can make use of. Before version 2.8, when there were no rules available to be downloaded for a certain country, after a check Corona-Warn-App displayed 'Certificate Valid' and explained in the following text, that for this country no rules were provided/could have been checked. 'Certificate Valid' referred to 'Technical Validity' in this case (more information on what the 'Technical Validity' is can be found in <a href='#eu_dcc_validity' target='_blank' rel='noopener noreferrer'>this</a> FAQ entry).",
+                            "However, we understood that this result may be misinterpreted, for example when an incomplete vaccination was checked, the wrong conclusion could have been drawn, that this certificate warrants entry without restrictions. We clarified this with version 2.8: In case a member country did not provide entry rules in machine-readable format for Corona-Warn-App, the headline now states 'Certificate Cannot Be Validated', whereas the following texts stayed the same. This does <b>not</b> mean that your certificate is not valid or that you are not allowed to enter the country you checked your certificate against. In this case, please verify your status with information that you find in <a href='https://reopen.europa.eu/en' target='_blank' rel='noopener noreferrer'>https://reopen.europa.eu/en</a>. Further information on this change can be found in the second paragraph of this blog post: <a href='../blog/2021-08-25-cwa-version-2-8/' target='_blank' rel='noopener noreferrer'>Corona-Warn-App version 2.8 displays names in certificates in standardized characters</a>.<hr>",
+                            "<b>Why does the CovPass app show a different result when I check entry rules for a country that did not provide entry rules in machine-readable format yet?</b>",
+                            "The validity check for entry rules was first introduced in Corona-Warn-App with version 2.6. When this validity check was developed for CovPass, Corona-Warn-App's result screen served as a template for CovPass' current implementation. We are working on a common soloution for both apps."

```

Comment:
soloution => solution (typo)

Reply:
@Ein-Tim 
I can't see soloution => solution (typo) corrected.

Reply:
@MikeMcC399 

Sorry, I forget this 😅

Fixed with [`47d3b95` (#1683)](https://github.com/corona-warn-app/cwa-website/pull/1683/commits/47d3b95319e20f981057f4ae480356f3def8397c).